### PR TITLE
Teams

### PR DESCRIPTION
**Pull request by emersion**
Merges #550 and #547.

* [x] Add `LICENSE.md`
* [x] Add automated Web Team page.
* [x] Add automated Desktop Team page.
* [ ] Add Teams index.
* [ ] Production-grade CSS.
* [ ] Code is duplicated between teams pages

Preview: 
* http://beta.elementary.io/branch/teams/teams/web
* http://beta.elementary.io/branch/teams/teams/desktop

### CODE REVIEW

**Comment by emersion** [comment]
@lewisgoddard 
screenshot from 2015-04-19 15-22-02

**Comment by lewisgoddard** [comment]
@emersion Maybe we can trim names that go over twelve characters and add a `&hellip;`

**Comment by emersion** [comment]
@lewisgoddard Not possible with CSS?

**Comment by emersion** [comment]
- [ ] Code is duplicated between teams pages

**Comment by danrabbit** [comment]
Is it possible to get real names instead of screen names?

**Comment by emersion** [comment]
@danrabbit For Launchpad contributors, it is possible (it is the `display_name` key in the JSON file). For Github contributors, it's not.

**Comment by danrabbit** [comment]
To be honest it looks really amateur to have screen names instead of real names. We would need some way to match those on our side then

File: styles/main.css [hunk 1]
```diff
@@ -413,22 +413,34 @@ footer ul li a:focus {
     margin-bottom: 16px;
 }
 
-@media (min-width: 444px) {
+@media (min-width: 1200px) {
```
Comment (cassidyjames) [inline styles/main.css]: All this grid re-working will likely have consequences across the whole site. I don't think "fixing" it in this branch is really appropriate. Could you propose PRs with these changes, explaining why you're changing them, before this branch is merged?

**Comment by lewisgoddard** [comment]
While I still like the idea of automated teams pages, there are clearly a couple of technical problems with the implementation. We've already created a [LICENSE.md](https://github.com/elementary/mvp/blob/master/LICENSE.md), and have started linking to [GitHub Contributors](https://github.com/elementary/mvp/graphs/contributors) as the "Web Team" page. If a similar thing is done for desktop, the only remaining issue is layout, which is a separate bug altogether, #332 in fact.

**Comment by lewisgoddard** [comment]
I'm going to close this for now, as I think it's getting out of hand. Small, testable pull requests are much better than behemoths like this.

**Pull request closed without merging**